admin-dev/themes/new-theme/js/pages/order/OrderViewPageMap.js

```diff
@@ -141,10 +141,12 @@ export default {
       save: '#cancel_product_save',
       partialRefund: 'button.partial-refund-display',
       standardRefund: 'button.standard-refund-display',
+      cancel: 'button.cancel-product-display',
```

Comment:
```suggestion
      cancelProducts: 'button.cancel-product-display',
```
To avoid thinking it's a form cancel button

Reply:
done